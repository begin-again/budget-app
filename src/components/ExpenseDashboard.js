import React from 'react';
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';

const ExpenseDashboard = () => {
  const template =
    <div>
      <ExpenseListFilters />
      <ExpenseList />
    </div>
        ;
  return template;
};

export default ExpenseDashboard;
