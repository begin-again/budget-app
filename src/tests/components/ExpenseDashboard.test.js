/* eslint-env jest */
import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboard from '../../components/ExpenseDashboard';

describe('Expense Dashboard Component', () => {
  it('should render', () => {
    const wrapper = shallow(<ExpenseDashboard />);

    expect(wrapper).toMatchSnapshot();
  });
});
