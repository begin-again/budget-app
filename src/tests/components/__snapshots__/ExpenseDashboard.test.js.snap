// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expense Dashboard Component should render 1`] = `
<div>
  <[object Object] />
  <[object Object] />
  <[object Object] />
</div>
`;
